Fix: review the partnership page (#273)
pyOpenSci/software-peer-review: partners/scientific-communities.md

# Partnerships with scientific Python Communities

:::{toctree}
:hidden:
:caption: Our Partners

About Our Partnerships <self>
JOSS <joss>
Pangeo <pangeo>
:::

:::{figure-md} pyopensci-3-pillars

> <img src="../images/pyopensci-pillars-flower.png" alt="Flower" width="500">

Author: fixed alt tag!

> pyOpenSci has three core programs. Community partnerships is a program that allows domain specific scientific Python communities to leverage our peer review process rather than creating their own process. Through this partnership maintainers can become affiliated with both a domain specific community, published in the Journal of Open Source Software and also become a part if the pyOpenSci community.

Reviewer: ```suggestion
pyOpenSci has three core programs: program one, program two, program three. Community partnerships allow domain-specific scientific Python communities to leverage the pyOpenSci peer review process rather than creating their own process. Through this partnership, maintainers can become affiliated with a domain-specific community, published in the Journal of Open Source Software (JOSS) and join the pyOpenSci community.
```

Author: @kierisi your in place text change makes the text a bit confusing with program one, program two etc. are you suggesting that  i list the name of each program there? or we could add numbers before each section? i just wouldn't want generic text.

Author: if you wanted me to insert the program names a nice approach could be to only edit the text inline where you are confident in actually changing the text (if i merge this the text will now say program one, program two, etc) . then add a note to the suggested change that says - i'd spell out the program names or something like that. because as is i can't merge this (unless i am mis understanding the intent that is!)

Author: ok i have. just rewritten this all alt tag and caption so it's more clear. resolving!

:::

> Numerous science domain-specific scientific Python communities are developing and maintaining their own peer review

Reviewer: ```suggestion
Numerous domain-specific scientific Python communities are developing and maintaining their own peer review
```

> processes to track and recommend packages that relevant to their science domain.

Reviewer: ```suggestion
processes to track and recommend packages relevant to their specific area(s) of expertise.
```

However, running a high quality peer review process takes time and resources.

> pyOpenSci offers a program where communities can partner with us, and leverage our peer review process, which is

Reviewer: ```suggestion
pyOpenSci offers a program where communities can partner with us and leverage our peer review process, which is
```

> built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a community-specific

Reviewer: ```suggestion
built on community-driven packaging guidelines and standards. Through this partnership, we enhance our standard review process with a custom review process
```

Author: i really appreciate the concept here but i'm slightly worried about the word "custom" which might make it sound like we can totally modify our process. if there is a better word to suggest adding a layer on top of an existing structure i think that would be better. Community specific related to adding guidelines specific to that community but the process is still the same (with that additional layer). i know this is very nuanced but i wouldn't want someone to think we can totally customize our review process as that would be too much work to maintain. i'll think about other words or perhaps you have ideas too!! i may merge this before you are back only because of astropy moving forward now BUT  we can also always go back in and modify!

Author: ok i know where this is from now. and i'll think about what words we want to use. it's a custom step rather than a custom review process. maybe i'll adjust the graphic language and finesse this...

> step that adds your specific requirements for package affiliation.

Reviewer: ```suggestion
that integrates your specific requirements for package affiliation.
```

:::{figure-md} pyopensci-partners

<img src="../images/peer-review-partners-process.png" alt="." width="500">

> Through this partnership we add an additional layer of checks to our peer review process that align with a domain specific communities needs. Often these checks are around packages using existing core functionality of community tools, and adhering to community specific data structure formats.

Reviewer: ```suggestion
Through this partnership, we add an additional layer of checks to the pyOpenSci peer review process that align with a domain-specific community's needs. Often these checks are around packages using existing core functionality of community tools and adhering to community-specific data structure formats.
```

:::

:::{figure-md} pyopensci-joss-community

> <img src="../images/pyos-partnerships-peer-review.png" alt="" width="400">

Author: NOTE: missing an alt tag for the new graphic!!

Author: added!

> Through this partnership, we add an additional layer of checks to the pyOpenSci peer review process that align with a domain-specific community's needs. Often these checks are around packages using existing core functionality of community tools and adhering to community-specific data structure formats.

Author: Ahhh @kierisi i see where that word customized came from. i used it in this graphic. i'll revisit that word as it's bothering me now. maybe customized review step? rather than the entire review being custom?

> partnership with us means that you will always have support if you run into

Reviewer: ```suggestion
partnership with pyOpenSci means that you will always have support if you run into
```

Reviewer: can we be more specific on what "support" looks like? maybe something like "always have access to support from our community of Python experts" etc.

Author: great feedback! i've modified it to this:

> Our vibrant community is filled with experts in the Python packaging space. A
> partnership with us means that you will always have support from our diverse community of Pythonistas if you run into
> Python packaging and software maintenance challenges.

Python packaging issues.

:::{admonition} What's in the works
:class: note

> We are currently enhancing our peer review process and working together on:

Reviewer: ```suggestion
We are currently enhancing our peer review process and collaborating on:
```

Reviewer: it was unclear who the "we" is in "working together"

Author: totally - i've changed it as follows:

> - Adding an editorial bot to our peer review infrastructure to streamline our review processes.
> - Collecting package metrics to track health and also understand maintenance status.

> -Adding an editorial bot to our peer review infrastructure to further streamline our processes.

Reviewer: ```suggestion
- Adding an editorial bot to our peer review infrastructure to further streamline our processes.
```

> - Collecting metrics around package maintenance to track health and to help identify when packages become unmaintained.

Reviewer: ```suggestion
- Collecting metrics around package maintenance to track package health and to help identify when packages become unmaintained.
```

Reviewer: both of these processes are important, but they sound kind of...sterile? slightly negative? in other words, if I'm interested in collaborating with pyOS, these make it sound like:
- pyOS doesn't have a fully worked-out process
- packages are unhealthy

the first one is fine, but I think rewording the second to be more positive, along the lines of "to track package health and ensure all pyOS packages are well-maintained"

Reviewer: I'm also wondering if there's anything else we could talk about here that would be a direct benefit to potential partners - something they'd see and go "oh, that's interesting!" or "we could really use that!"

Author: A bit of context:

Every partner community i've talked to has a pain point of tracking these things! that is what they want. currently many have these manual badges that quickly get out of date. so us automating it and working together as a community is that "we could really use that" feature that everyone wants! the bot is more for us / review to ensure consistency. it's so easy to forget steps such as adding labels, etc.  

however - i agree that it could read as  unhealthy AND we don't have a process yet - part of the collab will be figuring out what these checks look like. 

We already did work together on our archiving policy in the peer review guide with a few communities. So the health metrics will be similar i think and will be a learning process! 

with that said i've rewritten as follows to try to address that "it sounds negative" vibe it was giving

> 
> pyOpenSci is enhancing our peer review process and working on:
> 
> - Adding an editorial bot to our peer review infrastructure to streamline our review processes.
> - Collecting package metrics to track health and also understand maintenance status.

we could consider using a different word than health. i'm just not sure what word to insert!

Reviewer: @lwasser I don't think there's an issue with the word health, but rather "help identify when packages become unmaintained" - so what I'm thinking is more "help identify when packages are in need of maintenance" or something to indicate that we're keeping an eye on the issue. and if we offer assistance on intervention (finding new maintainers, etc.) mentioning that as well!
